device/celestica/x86_64-cel_silverstone-r0/sonic_platform/sfp.py

```diff
@@ -270,7 +269,7 @@ def __convert_string_to_num(self, value_str):
             return 'N/A'
 
     def __is_host(self):
-        return os.system(self.HOST_CHK_CMD) == 0
+        return subprocess.call(self.HOST_CHK_CMD) == 0
```

Comment:
Please consider changing the is_host function implementation as suggested above.